### DISCUSSION TITLE
Clipboard integration

### DISCUSSION THREAD

**Discussion by martinbaines** [kasmtech/KasmVNC#248] [Q&A]
In the docs it said that for Chromium based browsers, clipboard integration works and there should be no need to used the pit stop in the "clipboard!" option menu.
I have been using vnc but installed myself on Debian and Ubuntu servers directly, and indirectly on various docker apps that use kasmvnc to expose a UI, and have never had the integration work (tried with both Chromium, and Chrome). Is there something that needs to be done to get it to work?
It is not a huge deal but thought it worth checking if there is some hidden option needed for this to work.

**Comment by BlakeFL** [comment]
I also have this issue with Brave which I believe is chromium based.  Is there a browser security setting we need to update?
